war/src/main/java/io/lumeer/engine/rest/OrganizationService.java

```diff
@@ -0,0 +1,389 @@
+/*
+ * -----------------------------------------------------------------------\
+ * Lumeer
+ *  
+ * Copyright (C) 2016 - 2017 the original author or authors.
+ *  
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -----------------------------------------------------------------------/
+ */
+package io.lumeer.engine.rest;
+
+import io.lumeer.engine.api.data.DataDocument;
+import io.lumeer.engine.api.exception.OrganizationAlreadyExistsException;
+import io.lumeer.engine.api.exception.UserAlreadyExistsException;
+import io.lumeer.engine.controller.OrganizationFacade;
+
+import java.io.Serializable;
+import java.util.List;
+import java.util.Map;
+import javax.enterprise.context.RequestScoped;
+import javax.inject.Inject;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+/**
+ * @author <a href="mailto:mat.per.vt@gmail.com">Matej Perejda</a>
+ * @author <a href="alica.kacengova@gmail.com">Alica Kačengová</a>
+ */
+@Path("/organizations")
+@RequestScoped
+public class OrganizationService implements Serializable {
+
+   private static final long serialVersionUID = 3125094059637285633L;
+
+   @Inject
+   private OrganizationFacade organizationFacade;
+
+   // TODO: RequestScoped?
```

Comment:
There is no need to create a new instance of this class with every request so it can be `ApplicationScoped`.